rabbitmq-utils/README.md

```diff
@@ -0,0 +1 @@
+# docker-rabbit-util
```

Comment:
Now that you mention it, it would be nice to know which utils we have here :)

Reply:
Added some info to the Readme